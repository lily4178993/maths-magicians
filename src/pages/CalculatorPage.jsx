import React from 'react';
import { Calculator } from '../components';

const CalculatorPage = () => (
  <section className="section-calculator">
    <h2 className="section-title fade-in-left">Let&apos;s do some maths!</h2>
    <Calculator />
  </section>
);

export default CalculatorPage;
