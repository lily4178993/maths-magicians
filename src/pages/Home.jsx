import React from 'react';

const Home = () => (
  <section className="section-home fade-in-top">
    <h2 className="section-title fade-in-left">Welcome to our page!</h2>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure, voluptatum
      eum! Tempora itaque saepe iure sed aut magni minima soluta laborum,
      tenetur voluptas eos, dignissimos, eligendi doloremque quo perspiciatis
      natus!
    </p>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure, voluptatum
      eum! Tempora itaque saepe iure sed aut magni minima soluta laborum,
      tenetur voluptas eos, dignissimos, eligendi doloremque quo perspiciatis
      natus!
    </p>
  </section>
);

export default Home;
