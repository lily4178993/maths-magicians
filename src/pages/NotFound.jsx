import React from 'react';
import { Link } from 'react-router-dom';

const NotFound = () => (
  <section className="noFound fade-in-top">
    Page not found. Go to
    <Link to="/">Home</Link>
    page
  </section>
);

export default NotFound;
