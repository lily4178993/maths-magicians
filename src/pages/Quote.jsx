import React from 'react';
import { Quotes } from '../components';

const Quote = () => (
  <section className="section-quotes fade-in-top"><Quotes /></section>
);

export default Quote;
