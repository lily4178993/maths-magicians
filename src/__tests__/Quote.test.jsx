import React from 'react';
import { render } from '@testing-library/react';
import { Quote } from '../pages';

test('Quote component snapshot', () => {
  const { asFragment } = render(<Quote />);
  expect(asFragment()).toMatchSnapshot();
});
