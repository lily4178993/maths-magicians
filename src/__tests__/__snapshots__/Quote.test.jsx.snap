// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote component snapshot 1`] = `
<DocumentFragment>
  <section
    class="section-quotes fade-in-top"
  >
    <div
      class="loading-message"
    >
      Loading...
    </div>
  </section>
</DocumentFragment>
`;
